anna/my-app/src/components/searchForm/SearchForm.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import './searchForm.css';
+import Button from '../../components/button';
+
+interface ISearchFormProps {
+  onSubmit({ }: ISearchFormState): void;
+}
+
+export interface ISearchFormState {
+  value: string;
+  searchBy: string
+}
+
+class SearchForm extends React.Component<ISearchFormProps, ISearchFormState> {
+  state: ISearchFormState = {
+    value: '',
+    searchBy: 'title'
+  };
+
+  changeValue = (e: React.ChangeEvent<HTMLInputElement>) => this.setState({ value: e.target.value });
+  handleOptionChange = (e: React.ChangeEvent<HTMLInputElement>) => this.setState({ searchBy: e.target.value });
```

Comment:
according to the task you have to clear an input on option changes

Reply:
done